app/javascript/v3/components/Form/Textarea.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <with-label
+    :label="label"
+    :name="name"
+    :has-error="hasError"
+    :error-message="errorMessage"
+  >
+    <textarea
+      :id="name"
+      :name="name"
+      autocomplete="off"
+      :required="required"
+      :placeholder="placeholder"
+      :data-testid="dataTestid"
+      :value="value"
+      :rows="rows"
+      :class="{
+        'focus:outline-red-600 outline-red-600': hasError,
+        'dark:outline-slate-600 dark:focus:outline-woot-500 outline-slate-200 focus:outline-woot-500':
+          !hasError,
+        'px-3 py-3': spacing === 'base',
+        'px-3 py-2': spacing === 'compact',
+        'resize-none': !allowResize,
+      }"
+      class="block w-full border-none rounded-xl shadow-sm appearance-none outline outline-1 focus:outline focus:outline-2 text-slate-900 dark:text-slate-100 placeholder:text-slate-400 dark:bg-slate-800"
```

Comment:
@nithindavid Do we need `focus:outline` here? Already added `focus:outline-2` and `outline`

Reply:
`outline` is for `outline-style: solid;` and `outline-2` is the width of the outline.

Reply:
Why do we change the width of the outline on focus>

Reply:
This is to have a distinguished focus effect.

Reply:
Why is not ring working on production?